solidity/random-datasource/relaxed-commit/relaxedRandom.sol

```diff
@@ -1,76 +1,137 @@
-/*
-   Oraclize "relaxed" random-datasource example
-
-   This contract uses the random-datasource to securely generate off-chain N random bytes
-
-    Lower security guarantees than default setting
-*/
-
-pragma solidity >= 0.4.11 < 0.5;
-
-import "github.com/oraclize/ethereum-api/oraclizeAPI_0.4.sol";
-
-contract RandomExample is usingOraclize {
-    
-    event newRandomNumber_bytes(bytes);
-    event newRandomNumber_uint(uint);
-
-    function RandomExample() {
-        oraclize_setProof(proofType_Ledger); // sets the Ledger authenticity proof in the constructor
-        update(); // let's ask for N random bytes immediately when the contract is created!
+/**
+ * @notice  Provable Random Datasource Example - Relaxed Commit Version
+ *
+ *          This contract uses the random-datasource to securely generate
+ *          off-chain random bytes. The relaxed commit parameters result in a
+ *          greater chance of a passing proof in exchange for lower security
+ *          guarantees.
+ *
+ *          The random datasource is currently only available on the
+ *          ethereum main-net & public test-nets (Ropsten, Rinkeby & Kovan).
+ *
+ */
+pragma solidity >= 0.5 < 0.6;
+
+import "github.com/oraclize/ethereum-api/oraclizeAPI_0.5.sol";
+
+contract RandomRelaxedExample is usingOraclize {
+
+    uint256 NUM_RANDOM_BYTES_REQUESTED = 7;
```

Comment:
since this is never re-assigned, may as well just be a constant as well, however, not anything important in my book.

Reply:
Yeah, should have though of that really while I was making it `SCREAMING_SNAKE_CASE`.

Will push it since it's like a 30s job.